tests/test_indexer.py

```diff
@@ -554,6 +556,66 @@ def test_multiple_schema_version_subscription_indexing_and_notification(self):
 
         self.delete_subscription(subscription_id)
 
+    def test_scrub_index_data(self):
+        with self.subTest("with schema version"):
+            'Extra fields are removed.'
+            manifest = read_bundle_manifest(self.blobstore, self.test_bucket, self.bundle_key)
+            index_data = create_index_data(self.blobstore, self.test_bucket, manifest)
+            index_data['files']['assay_json']['extra_top'] = 123
+            index_data['files']['assay_json']['extra_obj'] = {"something": "here", "another": 123}
+            index_data['files']['assay_json']['extra_lst'] = ["a", "b"]
+            index_data['files']['assay_json']['core']['extra_internal'] = 123
+            with self.assertLogs(logger, level="INFO"):
```

Comment:
What's this for? There are no assertions.

Reply:
It checks that a LOG message was produced. I'll add a regex to verify the log returned.